docs/bucket/notifications/README.md

```diff
@@ -163,6 +163,7 @@ The Minio server configuration file is stored on the backend in json format. The
 | `clientId` | _string_ | Unique ID for the MQTT broker to identify Minio |
 | `username` | _string_ | Username to connect to the MQTT server (if required) |
 | `password` | _string_ | Password to connect to the MQTT server (if required) |
+| `queueDir` | _string_ | Persistent store for events (offline) |
```

Comment:
It is not clear what is offline from this description. We could say something like, `Persistent store for events when MQTT broker is offline`

Reply:
changed it :+1: